dist/ramda.js

```diff
@@ -3263,6 +3246,35 @@
 
     /**
      * Converts an object into an array of key, value arrays.
+     * Only the object's own properties are used.
+     * Note that the order of the output array is not guaranteed to be
+     * consistent across different JS platforms.
+     *
+     * @func
+     * @memberOf R
+     * @category Object
+     * @sig {String: *} -> [[String,*]]
+     * @param {Object} obj The object to extract from
+     * @return {Array} An array of key, value arrays from the object's own properties.
+     * @example
+     *
+     *      R.toPairs({a: 1, b: 2, c: 3}); //=> [['a', 1], ['b', 2], ['c', 3]]
+     */
+    var toPairs = _curry1(function toPairs(obj) {
+        var pairs = [];
+        for (var prop in obj) {
+            if (Object.prototype.hasOwnProperty.call(obj, prop)) {
```

Comment:
@CrossEye (about Cherries): I was not planning to, but if there's any interest, then I'd love to do it. ... in fact I think I might do it anyway. I'm missing a personal project to work on currently.

edit: Sorry, I didn't realise I was writing in the wrong box :(